fix: Handles update of `mongodbatlas_backup_compliance_policy` as a create operation (#2480)
mongodb/terraform-provider-mongodbatlas: .changelog/2480.txt

```release-note:bug

> resource/mongodbatlas_backup_compliance_policy: Fixes issue where Update operation modified attributes which were not supposed to be updated"

Reviewer: ```suggestion
resource/mongodbatlas_backup_compliance_policy: Fixes an issue where the update operation modified attributes that were not supposed to be modified"
```
